### ISSUE TITLE
integration of backend using nodejs, expressjs, mongodb 

### ISSUE THREAD

**Issue by sejalrai23** [KaizenGirl1111/GSGrihSangini#21]
i would like to build signup and login api for user so that they can create account . please assign me this issue

**Comment by KaizenGirl1111** [comment]
@sejalrai23 assigned... sorry for late reply my exams were going on.. :)

**Comment by KaizenGirl1111** [comment]
@sejalrai23 any update

**Comment by sejalrai23** [comment]
yeah im working on it , what all are the fields i need to add , can you put them together here ?

**Comment by KaizenGirl1111** [comment]
@sejalrai23 could you check Issue#5 , a mongodb database was made by Pranavv213 you can check the discussion and things left that can be done

**Comment by ayush25102001** [comment]
@KaizenGirl1111 mam can I work on this issue

**Comment by KaizenGirl1111** [comment]
@ayush25102001 you can work on this one too! Good luck
